util/start_test

```diff
@@ -672,6 +672,7 @@ def set_up_environment():
         #   when the memkind package is being used (KNL, e.g.)
         # - any locale model other than 'flat' (because we'll need
         #   libnuma which is only available as a .so)
+        # MPF: this causes problems on my Ubuntu system...
```

Comment:
Yeah, please make an issue out of this or let's try to fix whatever is going wrong